tests/unit/lib/Marlin/MMU2/stubs/mmu2_log.cpp

```diff
@@ -13,9 +14,11 @@ void LogEchoEvent_P(const char *msg_P) {
 }
 
 void LogRequestMsg(const char *msg) {
+    assert((strlen(msg) + 1) < (uint8_t)mp::Protocol::MaxRequestSize() + 1);
```

Comment:
use REQUIRE() in unit tests and the related stubs - it is an "advanced" assert playing nicely with the unit test infrastructure

Reply:
Thanks for pointing that out.